test/controllers/tasks/statement.of.capital.controller.unit.ts

```diff
@@ -56,7 +57,40 @@ describe("Statement of Capital controller tests", () => {
   });
 
   describe("get tests", () => {
-    it("should navigate to the statement of capital page with NO buttons visible if share totals DO NOT match the shareholders'.", async () => {
+
+    it("should navigate to the statement of capital page with NO buttons visible, a share-capital-mismatch-warning and total-amount-unpaid-warning messages if Share totals DO NOT match and amount unpaid is NULL", async () => {
+      mockGetStatementOfCapitalData.mockResolvedValueOnce({
```

Comment:
I'd also add a test to do the same as this one but with amount unpaid set to undefined, so that we know the code will handle it being undefined.